src/frontend/web_application/src/components/form/Switch/index.jsx

```diff
@@ -1,31 +1,24 @@
 import React, { PropTypes } from 'react';
-import { v1 as uuidV1 } from 'uuid';
 import './style.scss';
 
-const Switch = ({ label, duplicateLabel, ...inputProps }) => {
-  const id = uuidV1();
-
-  return (
-    <div>
-      <div className="m-switch">
-        <input
-          type="checkbox"
-          className="m-switch__input"
-          id={id}
-          {...inputProps}
-        />
-        <label className="m-switch__paddle" htmlFor={id}>
-          <span className="show-for-sr">{label}</span>
-        </label>
-      </div>
-      { duplicateLabel && <label htmlFor={id} className="m-switch__label">{label}</label> }
-    </div>
-  );
-};
+const Switch = ({ label, id, ...inputProps }) => (
+  <div className="m-switch">
+    <input
+      type="checkbox"
+      className="m-switch__input"
+      id={id}
+      {...inputProps}
+    />
+    <label className="m-switch__paddle" htmlFor={id}>
+      <span className="show-for-sr">{label}</span>
+    </label>
+  </div>
+);
 
 Switch.propTypes = {
   label: PropTypes.string.isRequired,
-  duplicateLabel: PropTypes.bool,
+  id: PropTypes.string.isRequired,
```

Comment:
Oh okay, we have to use SwitchFieldGroup now. 

So you need to look for Switch component in the project and replace it by SwitchFieldGroup, else it will be "broken".